jb/fftw/time_delay_estimator_many.hpp

```diff
@@ -108,7 +112,8 @@ class time_delay_estimator_many {
     // Validate the input sizes.  For some types of timeseries the
     // alignment may be different too, but we only use the alignment
     // when the type of timeseries guarantees to always be aligned.
-    if (a.size() != tmpa_.size() or b.size() != tmpa_.size()) {
+    if (jb::detail::element_count(a) != jb::detail::element_count(tmpa_) or
```

Comment:
You request the same **shape** in the constructor, and just the same element count here.  I think you want the same shape in both cases.